src/common/sonarcloud-v2/config.ts

```diff
@@ -1,6 +1,6 @@
 // When the user does not specify a specific version, these willl be the default versions used.
 const msBuildVersion = "6.2.0.85879";
-const cliVersion = "6.1.0.4477";
+const cliVersion = "6.2.1.4610";
```

Comment:
Also updated minor of previous versions

Reply:
we can keep an eye if anyone runs into file size issues